tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -121,10 +121,15 @@ var _ = Describe("odo devfile create command tests", func() {
 	})
 
 	Context("When executing odo create with devfile component type argument and --registry flag", func() {
-		It("should successfully create the devfile component", func() {
+		It("should successfully create the devfile component if specified registry is valid", func() {
 			componentRegistry := "DefaultDevfileRegistry"
 			helper.CmdShouldPass("odo", "create", "java-openliberty", "--registry", componentRegistry)
 		})
+
+		It("should fail to create the devfile component if specified registry is invalid", func() {
```

Comment:
Can you also validate the log message that you added?

Reply:
Yep, I can do that.